libnavigation-core/api/current.txt

```diff
@@ -0,0 +1,443 @@
+    method public com.mapbox.navigation.core.replay.route.ReplayRouteOptions build();
+    method public com.mapbox.navigation.core.replay.route.ReplayRouteOptions.Builder maxAcceleration(double maxAcceleration);
+    method public com.mapbox.navigation.core.replay.route.ReplayRouteOptions.Builder maxSpeedMps(double maxSpeedMps);
+    method public com.mapbox.navigation.core.replay.route.ReplayRouteOptions.Builder minAcceleration(double minAcceleration);
+    method public com.mapbox.navigation.core.replay.route.ReplayRouteOptions.Builder turnSpeedMps(double minSpeedMps);
+    method public com.mapbox.navigation.core.replay.route.ReplayRouteOptions.Builder uTurnSpeedMps(double uTurnSpeedMps);
+  }
+
+}
+
+package com.mapbox.navigation.core.sensors {
+
+  public final class SensorEventEmitter implements android.hardware.SensorEventListener {
+    ctor public SensorEventEmitter(android.hardware.SensorManager sensorManager);
+    method public void onAccuracyChanged(android.hardware.Sensor? sensor, int accuracy);
+    method public void onSensorChanged(android.hardware.SensorEvent event);
+    method public void start(com.mapbox.navigation.core.sensors.SensorOptions sensorOptions, kotlin.jvm.functions.Function1<? super android.hardware.SensorEvent,kotlin.Unit> eventEmitter);
+    method public void stop();
+  }
+
+  public final class SensorOptions {
+    ctor public SensorOptions(java.util.Set<java.lang.Integer> enabledSensorTypes, int signalsPerSecond, com.mapbox.navigation.core.sensors.SensorOptions.Builder builder);
+    method public java.util.Set<java.lang.Integer> component1();
+    method public int component2();
+    method public com.mapbox.navigation.core.sensors.SensorOptions.Builder component3();
+    method public com.mapbox.navigation.core.sensors.SensorOptions copy(java.util.Set<java.lang.Integer> enabledSensorTypes, int signalsPerSecond, com.mapbox.navigation.core.sensors.SensorOptions.Builder builder);
+    method public com.mapbox.navigation.core.sensors.SensorOptions.Builder getBuilder();
+    method public java.util.Set<java.lang.Integer> getEnabledSensorTypes();
+    method public int getSignalsPerSecond();
+    method public com.mapbox.navigation.core.sensors.SensorOptions.Builder toBuilder();
+  }
+
+  public static final class SensorOptions.Builder {
+    ctor public SensorOptions.Builder();
+    method public com.mapbox.navigation.core.sensors.SensorOptions build();
+    method public com.mapbox.navigation.core.sensors.SensorOptions.Builder enableAllSensors();
+    method public com.mapbox.navigation.core.sensors.SensorOptions.Builder enableSensors(java.util.Set<java.lang.Integer> sensorTypes);
+    method public com.mapbox.navigation.core.sensors.SensorOptions.Builder signalsPerSecond(int signalsPerSecond);
+  }
+
+}
+
+package com.mapbox.navigation.core.telemetry {
+
+  public final class TelemetryLocationAndProgressDispatcherKt {
+  }
+
+  public final class TelemetryUtilsKt {
```

Comment:
Could we avoid tracking this empty `public` class? 🤔 There are only top-level functions